firefox-ios/Client/Frontend/Browser/Tabs/Views/EmptyPrivateTabsView.swift

```diff
@@ -36,16 +36,14 @@ class EmptyPrivateTabsView: UIView {
 
     private let titleLabel: UILabel = .build { label in
         label.adjustsFontForContentSizeCategory = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .title2,
-                                                            size: UX.titleSizeFont)
```

Comment:
thanks for the update! If we aren't using the following font sizes, `titleSizeFont`, `descriptionSizeFont`, `buttonSizeFont`, could we remove these as well?

Reply:
Removed and code pushed.

Reply:
great, thank you! seems like there's a swiftlint error on line 53, otherwise these changes looks good!
![image](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/1422f88b-b369-48ca-b08b-fdbaace3c0f1)

Reply:
Swiftlint error fixed.